compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -95,6 +95,14 @@ def AMDAIELoweringStrategy :
   ];
 }
 
+def AMDAIELowerToUKernels :
+    Pass<"iree-amdaie-lower-to-ukernels", ""> {
+  let summary =
+      "Separate out parts of the IR that lower to a micro-kernel";
```

Comment:
It would be good to have a bit more in the summary about what level the micro-kernels are matching at. Is it a whole operation, or just the inner-most loop of the operation? Maybe it's not decided yet, based on the matmul vs mmt4d question earlier.